src/components/YourCompany/YourCompany.vue

```diff
@@ -81,7 +86,7 @@
             <div class="actions mr-4">
               <!-- TODO: only show buttons for named company -->
               <v-btn
-                v-if="companyNameChanges"
+                v-if="companyNameChanges || (isAlteration() && hasBusinessNameChanged)"
```

Comment:
`isAlteration()` threw me for a minute.
I initially wondered why it wasn't a getter (but not a big deal).
Then I wondered what it means... I suggest this be renamed to "isAlterationPage" (or ...Route or ...View).

Reply:
Will refactor a bit!

Reply:
Updated in next commit 👍 
Changed all the instance of isAlteration and isCorrection to `isAlterationView`